chall03/mtuomine.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const baseURL = "https://chall03.hive.fi/"
+
+func main() {
+	client := http.Client{Timeout: time.Second * 1}
+
+	// First request
+	res, err := client.Get(baseURL)
+	checkErr(err)
```

Comment:
check error in Go can be repetitive, and breaking the DRY concept, but you should avoid to create function like this to handle your errors.

Some error should panic/quit, some should just raise a warning or print debug.